2017/05/09 10:48:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\wamp64\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\wamp64\www\projet_new\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projet_new/index.php?r=gii
---
2017/05/09 10:52:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\wamp64\www\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\wamp64\www\projet_new\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/projet_new/?r=gii
---
2017/05/09 10:56:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'prozzl_test'' in C:\wamp64\www\yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\wamp64\www\yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\wamp64\www\yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\wamp64\www\yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\wamp64\www\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\wamp64\www\yii\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\wamp64\www\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\wamp64\www\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\wamp64\www\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\wamp64\www\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\wamp64\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\wamp64\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\wamp64\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\wamp64\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\wamp64\www\projet_new\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/projet_new/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/projet_new/index.php?r=gii/default/index
---
2017/05/09 12:08:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "inscription".' in C:\wamp64\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\wamp64\www\yii\framework\web\CController.php(270): CController->missingAction('inscription')
#1 C:\wamp64\www\yii\framework\web\CWebApplication.php(282): CController->run('inscription')
#2 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/inscriptio...')
#3 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\wamp64\www\prozzl\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/prozzl/index.php?r=site/inscription
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
<<<<<<< HEAD
2017/05/09 14:45:49 [error] [exception.CException] exception 'CException' with message 'Property "Utilisateur.id_adresse" is not defined.' in C:\wamp\www\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id_adresse')
#1 C:\wamp\www\yii\framework\validators\CRequiredValidator.php(58): CActiveRecord->__get('id_adresse')
#2 C:\wamp\www\yii\framework\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(Utilisateur), 'id_adresse')
#3 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(Utilisateur), NULL)
#4 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(808): CModel->validate(NULL)
#5 C:\wamp\www\prozzl\protected\components\UserIdentity.php(61): CActiveRecord->save()
#6 C:\wamp\www\prozzl\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#7 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#8 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#9 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#10 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#11 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#12 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#16 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#17 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 14:55:04 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 15:36:45 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 15:37:26 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
=======
2017/05/09 12:31:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "inscription".' in C:\wamp64\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\wamp64\www\yii\framework\web\CController.php(270): CController->missingAction('inscription')
#1 C:\wamp64\www\yii\framework\web\CWebApplication.php(282): CController->run('inscription')
#2 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/inscriptio...')
#3 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\wamp64\www\prozzl\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/prozzl/index.php?r=site/inscription
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 12:31:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "inscription".' in C:\wamp64\www\yii\framework\web\CController.php:483
Stack trace:
#0 C:\wamp64\www\yii\framework\web\CController.php(270): CController->missingAction('inscription')
#1 C:\wamp64\www\yii\framework\web\CWebApplication.php(282): CController->run('inscription')
#2 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/inscriptio...')
#3 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\wamp64\www\prozzl\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/prozzl/index.php?r=site/inscription
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 12:35:48 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "inscription".' in C:\wamp64\www\yii\framework\web\CController.php:878
Stack trace:
#0 C:\wamp64\www\yii\framework\web\CController.php(782): CController->renderPartial('inscription', Array, true)
#1 C:\wamp64\www\prozzl\protected\controllers\SiteController.php(174): CController->render('inscription', Array)
#2 C:\wamp64\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionInscription()
#3 C:\wamp64\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\wamp64\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\wamp64\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\wamp64\www\yii\framework\web\CWebApplication.php(282): CController->run('Inscription')
#7 C:\wamp64\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/Inscriptio...')
#8 C:\wamp64\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\wamp64\www\prozzl\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/prozzl/index.php?r=site/Inscription
>>>>>>> 1aa2c5abd7f4ea50b9823f20e17433119e4f0770
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 15:42:42 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 15:47:49 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 15:50:28 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
2017/05/09 15:50:34 [error] [exception.CDbException] exception 'CDbException' with message 'Table "utilisateur" does not have a column named "login".' in C:\wamp\www\yii\framework\db\schema\CDbCommandBuilder.php:697
Stack trace:
#0 C:\wamp\www\yii\framework\db\ar\CActiveRecord.php(1523): CDbCommandBuilder->createColumnCriteria(Object(CMysqlTableSchema), Array, '', Array, '`t`.')
#1 C:\wamp\www\prozzl\protected\components\UserIdentity.php(22): CActiveRecord->findByAttributes(Array)
#2 C:\wamp\www\prozzl\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\wamp\www\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\wamp\www\prozzl\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\wamp\www\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\wamp\www\prozzl\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/prozzl/index.php?r=site/login
HTTP_REFERER=http://localhost/prozzl/index.php?r=site/login
---
